bot/cogs/snakes.py

```diff
@@ -1,21 +1,35 @@
 # coding=utf-8
 import logging
-from typing import Any, Dict
+import re
 
-from discord.ext.commands import AutoShardedBot, Context, command
+import aiohttp
+import bs4
+import discord
+import html2text
+from discord.ext import commands
 
 log = logging.getLogger(__name__)
+API = 'http://en.wikipedia.org/w/api.php?format=json&redirects=1&action='
+
+rSENTENCE = re.compile(r'^.+?\.')
+rBRACK = re.compile(r'[[(].+?[\])]')
 
 
 class Snakes:
     """
     Snake-related commands
     """
 
-    def __init__(self, bot: AutoShardedBot):
+    def __init__(self, bot):
         self.bot = bot
+        self.aexec = bot.loop.run_in_executor
+        self.session = aiohttp.ClientSession(loop=bot.loop)
+        self.h2md = html2text.HTML2Text()  # TODO: use
+        self.base_query = API + 'parse&prop=text&page={}'
+        self.secs_query = API + 'parse&prop=sections&page={}'
+        self.img_query = API + 'query&titles={}&prop=pageimages&pithumbsize=300'
 
-    async def get_snek(self, name: str = None) -> Dict[str, Any]:
```

Comment:
Why did you remove the type hinting?

Reply:
Sorry. My partner was for a short while only able to use python 3.5 (and typing was added in 3.6) so I started removing hints preemptively for compatibility -- didn't take care to put them *all* back once he was able to get on 3.6, though.